src/main/java/com/spotinst/sdkjava/model/api/azure/statefulNode/ApiStatefulNodeTasksConfiguration.java

```diff
@@ -0,0 +1,75 @@
+package com.spotinst.sdkjava.model.api.azure.statefulNode;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiStatefulNodeTasksConfiguration implements IPartialUpdateEntity {
```

Comment:
adjustment and identifier parameters missing

Reply:
Same as signals.enabledAt . These are internal fields.

Reply:
enabledAt should not be used. This field is internal field which converts other param values to some code. So this field shouldn't be used.